ofl/justfonthuninn/METADATA.pb

```diff
@@ -0,0 +1,26 @@
+name: "Justfont Huninn"
+designer: "Justfont"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2025-03-18"
+fonts {
+  name: "Justfont Huninn"
+  style: "normal"
+  weight: 400
+  filename: "JustfontHuninn-Regular.ttf"
+  post_script_name: "JustfontHuninn-Regular"
+  full_name: "Justfont Huninn Regular"
+  copyright: "Copyright 2025 The Justfont-Huninn Project Authors (https://github.com/justfont/Justfont-Huninn)"
+}
+subsets: "chinese-traditional"
+subsets: "cyrillic"
+subsets: "japanese"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+primary_script: "Hani"
```

Comment:
@aaronbell Fontbakery is complaining that we have multiple CJK subsets. Please choose the most appropriate one.

Reply:
I've updated it to just be Trad Chinese. But I also wonder why we can only have one CJK subset...